### DISCUSSION TITLE
PrestaShopException right after clean installation with cli, There is no image type defined for "products"

### DISCUSSION THREAD

**Discussion by Zudjo** [PrestaShop/PrestaShop#32432] [Q&A] [answered]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

## Environment

- Windows 11 x64
- node 16.0.0
- npm 7.10.0
- ![image](https://user-images.githubusercontent.com/113335680/235920850-06c7719d-3a5b-4d40-8553-ce8e8107f896.png)

## Error

```
There is no image type defined for "products".
at line 364 in file src/Adapter/Image/ImageRetriever.php
```

![image](https://user-images.githubusercontent.com/113335680/235917402-f9ec61b9-da9c-40aa-8135-cfd133453cff.png)

### Expected behavior

No errors on clean installation.

### Steps to reproduce

- I cloned the PrestaShop repository with git, in `C:\xampp\htdocs`
- Moved out all the files from the auto generated `PrestaShop` folder, to simulate a real host
- Now `C:\xampp\htdocs` is the root folder of the project
- Ran `composer install`
- Compiled manually all 4 folders indicated in the doc [here](https://devdocs.prestashop-project.org/8/development/compile-assets/#compiling-assets)
- Ran `php .\install-dev\index_cli.php --domain=localhost --db_server=localhost --db_name=banane --db_user=root --db_password=  --prefix=  --email=gi*****@ec*********.it --password=A2************`

### PrestaShop version(s) where the bug happened

8.0.4

### PHP version(s) where the bug happened

8.0.28

**Comment by tesshsu** [comment, marked as answer]
Hi @Zudjo 
I got the same error once I re-install version 8
You could try to fix if you have access phpMyAdmin and go to {PREFIX}_image_type table 

```
INSERT INTO `{PREFIX}_image_type` (`id_image_type`, `name`, `width`, `height`, `products`, `categories`, `manufacturers`, `suppliers`, `stores`)
VALUES
	(10, 'large_banner', 960, 400, 0, 1, 0, 0, 0),
	(9, 'product_listing', 220, 220, 1, 1, 1, 1, 0),
	(8, 'category_default', 960, 350, 0, 1, 0, 0, 0),
	(7, 'home_default', 250, 250, 1, 0, 0, 0, 0),
	(6, 'large_default', 500, 500, 1, 0, 0, 0, 0),
	(5, 'medium_default', 300, 300, 1, 1, 1, 1, 0),
	(4, 'small_default', 125, 125, 1, 1, 1, 1, 0),
	(3, 'cart_default', 80, 80, 1, 0, 0, 0, 0);

```
You could find out from this[ issue 17539](https://github.com/PrestaShop/PrestaShop/issues/17539) 
Hope this could help

**Reply by Zudjo** [reply]
Thanks man, all I had to do was replace `{PREFIX}` with the prefix I chose for the tables, run it on the database, and it worked.

**Reply by addison74** [reply]
This works, thank you. It's a shame it was not fixed yet after an year from reporting.

**Comment by matks** [comment]
Hi @Zudjo you forgot to build assets. Use `make assets` see https://devdocs.prestashop-project.org/8/basics/installation/localhost/#download-dependencies

**Reply by Zudjo** [reply]
Hi @matks , I'm on windows, I don't have the `make` command.

As stated, I built all assets manually with `npm install` `npm run build` in each folder.

Any suggestion?

**Reply by tesshsu** [reply]
Hi @Zudjo 
Hence in Windows, just changed the build.sh --> LF unix ( by Windows it is CRLF)
Then you could run in Cygwin Terminal(or window Terminal) in your Prestashop root: $ make assets
![55](https://user-images.githubusercontent.com/3927152/236178070-4760c5df-6ee8-4d07-9a2c-0556b2b4e198.png)

**Comment by matks** [comment]
I converted this issue into a discussion because I'm quite sure there is no bug (see above comment) 😉

**Reply by Zudjo** [reply]
Thanks, sorry for publishing in the wrong section.

**Comment by addison74** [comment]
I am able to reproduce this issue with the latest Github version 8.1.7. May I asked why the Prestashop Team did not fix it after an year?

**Reply by kpodemski** [reply]
because there is probably nothing to fix, this error comes up if there was a problem with building the assets :)

the project uses CLI installation all over the places and this error doesn't exist
